stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -212,20 +212,29 @@ spec:
           ports:
             - name: api
               containerPort: 30000
+
+          {{- if .Values.pipelines.api.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-              path: /
-              port: api
-            initialDelaySeconds: 10
-            timeoutSeconds: 5
-            failureThreshold: 6
+              path: {{ default "/" .Values.pipelines.api.livenessProbe.path}}
+              port: {{ default "api" .Values.pipelines.api.livenessProbe.port}}
+            initialDelaySeconds: {{ default 20 .Values.pipelines.api.livenessProbe.initialDelaySeconds }}
+            timeoutSeconds: {{ default 10 .Values.pipelines.api.livenessProbe.timeoutSeconds }}
```

Comment:
Remove the use of default here. Place the default values in values.yaml

Reply:
+1

Reply:
done